src/status_im/chat/commands/impl/transactions.cljs

```diff
@@ -0,0 +1,133 @@
+
+(def assets-separator [react/view transactions-styles/asset-separator])
+
+(defview choose-asset [selected-event-creator]
+  (letsubs [assets [:wallet/visible-assets-with-amount]]
+    [react/view
+     [list/flat-list {:data                      (filter #(not (:nft? %)) assets)
+                      :key-fn                    (comp name :symbol)
+                      :render-fn                 (render-asset selected-event-creator)
+                      :enableEmptySections       true
+                      :separator                 assets-separator
+                      :keyboardShouldPersistTaps :always
+                      :bounces                   false}]]))
+
+(defn send-short-preview
+  [{:keys [content]}]
+  (let [parameters (:params content)]
+    [react/text {}
+     (str (i18n/label :command-sending)
+          (i18n/label-number (:amount parameters))
+          " "
+          (:asset parameters))]))
+
+(defview send-status [tx-hash outgoing]
+  (letsubs [confirmed? [:transaction-confirmed? tx-hash]
+            tx-exists? [:wallet-transaction-exists? tx-hash]]
+    [react/touchable-highlight {:on-press #(when tx-exists?
+                                             (re-frame/dispatch [:show-transaction-details tx-hash]))}
+     [react/view message-styles/command-send-status-container
+      [vector-icons/icon (if confirmed? :icons/check :icons/dots)
+       {:color           colors/blue
+        :container-style (message-styles/command-send-status-icon outgoing)}]
+      [react/view
+       [react/text {:style message-styles/command-send-status-text}
+        (i18n/label (cond
+                      confirmed? :status-confirmed
+                      tx-exists? :status-pending
+                      :else :status-tx-not-found))]]]]))
+
+(defview send-preview
+  [{:keys [content timestamp-str outgoing group-chat]}]
+  (letsubs [network [:network-name]]
+    (let [{{:keys [amount fiat-amount tx-hash asset] send-network :network} :params} content
+          recipient-name (get-in content [:params :bot-db :public :recipient])
+          network-mismatch? (and (seq send-network) (not= network send-network))]
+      [react/view message-styles/command-send-message-view
+       [react/view
+        [react/view message-styles/command-send-amount-row
+         [react/view message-styles/command-send-amount
+          [react/text {:style message-styles/command-send-amount-text
+                       :font  :medium}
+           amount
+           [react/text {:style (message-styles/command-amount-currency-separator outgoing)}
+            "."]
+           [react/text {:style (message-styles/command-send-currency-text outgoing)
+                        :font  :default}
+            asset]]]]
+        (when fiat-amount
+          [react/view message-styles/command-send-fiat-amount
+           [react/text {:style message-styles/command-send-fiat-amount-text}
+            (str "~ " fiat-amount " " (i18n/label :usd-currency))]])
```

Comment:
There is new logic in develop on how to show  correct fiat value https://github.com/status-im/status-react/commit/21595eede7628ba7b40885639ac69678a817ae79#diff-714969497f614cba3dfc82123b601492L99
I understand that your branch was probably created before those changes merged to develop. Please align with that logic.